### ISSUE TITLE
Too many tokens are being consumed due to repeated errors.

### ISSUE THREAD

**Issue by chambungg** [stackblitz/bolt.new#1568]
### Describe the bug

Hello,

I am reaching out regarding the significant number of tokens being consumed due to repeated errors. Despite numerous attempts to correct the same issues, there have been occasions where the problems persist. It’s concerning that an estimated half of the total tokens are being used for this purpose. Additionally, modifications are frequently being made to sections that were not indicated for revision, leading to further token consumption and unnecessary time spent correcting these unintended changes.

In comparison, adjustments through GPT or Claude are often more efficient. If possible, I would greatly appreciate compensation for the tokens expended in this way.

Thank you for your attention and understanding.

### Link to the Bolt URL that caused the error

all.new

### Steps to reproduce

Hello,

I am reaching out regarding the significant number of tokens being consumed due to repeated errors. Despite numerous attempts to correct the same issues, there have been occasions where the problems persist. It’s concerning that an estimated half of the total tokens are being used for this purpose. Additionally, modifications are frequently being made to sections that were not indicated for revision, leading to further token consumption and unnecessary time spent correcting these unintended changes.

In comparison, adjustments through GPT or Claude are often more efficient. If possible, I would greatly appreciate compensation for the tokens expended in this way.

Thank you for your attention and understanding.

### Expected behavior

Hello,

I am reaching out regarding the significant number of tokens being consumed due to repeated errors. Despite numerous attempts to correct the same issues, there have been occasions where the problems persist. It’s concerning that an estimated half of the total tokens are being used for this purpose. Additionally, modifications are frequently being made to sections that were not indicated for revision, leading to further token consumption and unnecessary time spent correcting these unintended changes.

In comparison, adjustments through GPT or Claude are often more efficient. If possible, I would greatly appreciate compensation for the tokens expended in this way.

Thank you for your attention and understanding.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by ciprianavr** [comment]
Yes we all confronted to those issues but aleays use undo button and do not let the tool to go further. Proper prompting differently all the time does the job. I consumed my first 10M learning, then next 15M developing 2 powerful platforms so worth the money and learning much more by doing it. We invest in the team behind it and they will solve everything for sure.
